packages/nimble-components/src/.eslintrc.js

```diff
@@ -31,6 +31,8 @@ module.exports = {
             message: 'Use a const object instead of an enum. See other types.ts files for examples.'
         }],
 
+        '@typescript-eslint/naming-convention': 'off',
```

Comment:
Can you give more context on this rule change? If we're opting out of this rule for good long term reasons we should add a comment explaining them. If this is a temporary suppression to limit the scope of pulling in a new ruleset we should file a tech debt issue to revisit it and link to that issue in a comment.

Reply:
You can just create an issue with the [tech debt issue template](https://github.com/ni/nimble/issues/new/choose) and leave it unassigned. We'll triage it and assign it once the PR goes in.

Reply:
#807